src/main/java/edu/wisc/my/portlets/feedback/beans/Feedback.java

```diff
@@ -67,6 +67,11 @@
     private String profile;
     private String referrer;
     
+    //Customizable Links
+    private String chatLink;
+    private String callLink;
+    private String howToLink;
+    
```

Comment:
I agree that modeling characteristics of the organization in the Feedback form object is not ideal. It would be cleaner to model aspects of the organization or tenant in an `Organization` or `Tenant` object, or sure, as simply stuff in the response model.